src/Command/Anonymization/AnonymizeCommand.php

```diff
@@ -218,14 +218,17 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             }
 
             $this->doAnonymizeDatabase();
-
-            if (!$this->doAnonymizeCurrentDatabase && !$this->doBackupAndRestoreInitial) {
-                $this->doBackupAnonymizedDatabase();
-            }
+            $this->doBackupAnonymizedDatabase();
```

Comment:
I add a fix: conditions were wrong here, following what's said in command description, anonymized database should always be backed up.

Reply:
The description:

>  Anonymize a given backup file or the local database.
> 
>                 This command will successively perform these steps:
>                     1/ Backup the local database,
>                     2/ Restore the given backup file,
>                     3/ Anonymize the data from the given backup file,
>                     4/ Backup the newly anonymized database by overwritting the given backup file,
>                     5/ Restore your database to its original state from the backup produced at step 1.
> 
>                 If called with the --local-database option, step 2 is skipped.
>                 If called with the --no-restore option, step 1 and 4 are skipped.